CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog for jasminewd2
 
+# 0.0.10
+
+- ([ff2e624](https://github.com/angular/jasminewd/commit/ff2e624159344cd83b04c6a6648334ba12e78ea6))
+  fix(webdriver): Pass in the control flow.
```

Comment:
Would be nice to mention that this is a breaking change, just in case we have any users out there other than Protractor (and for ourselves in 3 months).